docs/scully-lib-core.md

```diff
@@ -0,0 +1,107 @@
+# Scully Core
+
+## Idle Monitor
+
+Scully implements the `IdleMonitorService` to hook into Zonejs. When angular goes idle (**more precise, when all outgoing HTTP-requests are settled**)
+Scully trigger Puppeteer, and knows when it's ready to render.
+
+Without this process, we have to resort to a (25 seconds) timeout. this is both slower and unprecise, some pages need even more time.
+
+`IdleMonitorService` is available in `ScullyLibModule`.
+
+## Router Service
+
+`ScullyRoutesService` is an service to provide to the user access to certains methods and observables to know
+the routes rendered by Scully.
+
+- allRoutes\$
```

Comment:
Take this out, it will be available, but undocumented. Users should be using other options. (the allroutes$)